src/book/chapter.rs

```diff
@@ -1,24 +1,44 @@
-use std::path;
+use std::path::{Path, PathBuf};
+use book::metadata::Author;
 
 /// The Chapter struct holds the title of the chapter as written in the SUMMARY.md file,
 /// the location of the markdown file containing the content and eventually sub-chapters
+
+/// TODO use in template: author, description, index, class
+
 #[derive(Debug, Clone)]
 pub struct Chapter {
+    /// The title of the chapter.
     title: String,
-    file: path::PathBuf,
+    /// Path to chapter's markdown file.
+    file: PathBuf,
+
+    /// TODO The author of the chapter, or the book.
+    author: Author,
+    /// TODO The description of the chapter.
+    description: String,
+    /// TODO Index number of the chapter in its level. This is the Vec index + 1.
+    index: i32,
```

Comment:
What do you plan to do with this field? 
If this is just `Vec::len + 1`, what is the purpose of keeping it in the struct? By duplicating indices you add a synchronisation complexity. For example, if you add a chapter in the middle you need to update all the `index` fields.